.env.example

```diff
@@ -0,0 +1 @@
+PARALLEL_RUN=
```

Comment:
isn't the `.env.dist` the actual example file already? Why do we need this `.env.example` file?

Reply:
> isn't the .env.dist the actual example file already?

I will deleted it's not necesary
https://github.com/Chemaclass/bashunit/blob/4c6aa107d87d52937d6bc10a3ae545253db9dcda/bashunit#L9

Reply:
Done